spec/integration/manifest.spec.yml

```diff
@@ -577,6 +585,12 @@ spec:
 
 # JSON patches for newer versions should be placed on top
 versions:
+  - before: 3.3.2
+    patch:
+      - op: remove
+        path: "/definitions/deployment_modes/properties/agentless/properties/is_default"
+      - op: remove
+        path: "/definitions/deployment_modes/properties/agentless/properties/resources"
```

Comment:
Setting added in https://github.com/elastic/package-spec/pull/853, but we didn't add a version patch there.